sample/tests/test_validation.py

```diff
@@ -286,10 +160,6 @@ def test_ensure_payment_repository_failure() -> None:
     """Test convenience function with invalid payment repository"""
     # Create a mock that's definitely missing required methods
     invalid_repo = MagicMock()
-    # Remove all methods that might be auto-created by MagicMock
-    for method_name in ["process_payment", "get_payment"]:
-        if hasattr(invalid_repo, method_name):
-            delattr(invalid_repo, method_name)
```

Comment:
I don't think this was having any effect - those attributes won't be defined on a blank `MagicMock` without a spec?